apps/startup-scripts/README.md

```diff
@@ -135,6 +135,9 @@ export CONFIG="/path/to/worldserver.conf"
 export SESSION_MANAGER="tmux"  # none|auto|tmux|screen
 export SESSION_NAME="ac-world"
 
+# Interactive mode control
+export AC_DISABLE_INTERACTIVE="0"  # Set to 1 to disable interactive prompts (useful for non-interactive services)
```

Comment:
this is nitpick, but something I thought about. This is kind of an inverse option. Logically it would make more sense to rename it to AC_ENABLE_INTERACTIVE and have it default 1. as 1 = enable 0 = disable.

Does it matter? No, just something I thought about for way too long. xd

Reply:
Sorry but I can't do it now :( feel free to open a PR, I don't have anything against this change :)

Reply:
nah, as I said, nitpick. If it was in the actual core I would've cared more :D